### ISSUE TITLE
Go back to official firmware

### ISSUE THREAD

**Issue by matt3z** [TheCrypt0/yi-hack-v4#281]
Hi, is there a way to go back to the official firmware? I tried to download it from the Yi Official site, transferred it to the SDcard but the camera won't boot. Any suggestions?

**Comment by matt3z** [comment]
?

**Comment by matt3z** [comment]
Any solutions?

**Comment by mykl408** [comment]
Yes. same issue here.  If anyone can point to any "general" doc that would be helpful. It may not exist, which is fine, but if it does I would like to look at it. But as Matt said... if we can't get back to the original (yet crappy) firmware I won't be able to use it.

**Comment by balkurdi** [comment]
Same here

**Comment by matt3z** [comment]
Up

**Comment by matt3z** [comment]
Can someone give us a solution please?

**Comment by nicmela** [comment]
ded project
